SparkPost/2web2ui
# TR-2010 - Updates delete template modal styles to be much *much* scarier

[TR-2010](https://jira.int.messagesystems.com/browse/TR-2010)

### What Changed
- Refactored delete template modal to use pure props instead of editor context
- Updated styles for the delete template modal to be a lot scarier (matching the very frightening mocks)
- Refactored delete template modal tests to use React Testing Library
- Updated some old Enzyme tests to match the new implementation
- ...it maybe goes without saying, but some Prettier reformatting going on here

#### Before

![Screen Shot 2019-11-25 at 10 45 17 AM](https://user-images.githubusercontent.com/3613392/69655908-88ac4300-1045-11ea-9f7d-9a75dec42002.png)

#### After

![Screen Shot 2019-11-26 at 9 20 11 AM](https://user-images.githubusercontent.com/3613392/69655914-8b0e9d00-1045-11ea-84f7-419230f69abd.png)

### How To Test
- Go to `/templates` and delete a template from the 'Recent Activity' section and the `<table/>` - verify that the new styles are present *and* that deleting works as expected
- Click on a template in draft mode and use the editor actions in the top right and delete the template. Verify that the new styles are present and that deleting occurs as expected.
- Click on a template in published mode and use the editor actions in the top right and delete the template. Verify that the new styles are present and that deleting occurs as expected.
- Run Cypress tests for templates locally - `npm run test-e2e`

### To Do
